src/modules/solution/controllers/solution.controller.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { faker } from '@faker-js/faker';
+import { Test, TestingModule } from '@nestjs/testing';
+import { PrismaService } from '@src/modules/core/database/prisma/prisma.service';
+import { mockSolutionService } from '../../../../test/mock/mock-services';
+import { mockPrismaService } from '../../../../test/mock/mock-prisma-service';
+import { CreateSolutionRequestBodyDto } from '../dtos/create-solution-request-body.dto';
+import { SolutionEntity } from '../entities/solution.entity';
+import { SolutionService } from '../services/solution.service';
+import { SolutionController } from './solution.controller';
+
+describe('SolutionController', () => {
+  let solutionController: SolutionController;
+  let solutionService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [SolutionController],
+      providers: [
+        {
+          provide: SolutionService,
+          useValue: mockSolutionService
+        }
+      ]
+    }).compile();
+
+    solutionController = module.get<SolutionController>(SolutionController);
+    solutionService = mockSolutionService;
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  })
+
+  it('should be defined', () => {
+    expect(solutionController).toBeDefined();
+  });
+
+  describe('Create Lesson Solution', () => {
+    let createSolutionDto: CreateSolutionRequestBodyDto;
+    let memberId: number;
+    let solutionEntity: SolutionEntity;
+
+    beforeEach(() => {
+      memberId = faker.datatype.number();
+      createSolutionDto = {
+        lessonId: faker.datatype.number(),
+        description: faker.lorem.text(),
+        relatedLink: "https://github.com"
```

Comment:
```suggestion
        relatedLink: faker.internet.url()
```
찾아보니 faker에 URL에 관한 값도 제공을 해주더라구요